integration-auth/src/test/java/org/symphonyoss/integration/authentication/AuthenticationProxyImplTest.java

```diff
@@ -129,22 +119,22 @@ public void testInitWithoutSBEUrl() {
 
   @Test(expected = AuthUrlNotFoundException.class)
   public void testInitWithoutKeyManagerUrl() {
-    when(atlas.get(SESSION_AUTH_URL)).thenReturn("https://localhost:8444");
+    when(properties.getSessionManagerAuthUrl()).thenReturn("https://localhost:8444/sessionauth");
     proxy.init();
   }
 
   @Test
-  public void testInit() throws AtlasException {
-    when(atlas.get(SESSION_AUTH_URL)).thenReturn("https://localhost:8444");
-    when(atlas.get(KEY_AUTH_URL)).thenReturn("https://localhost:8444/relay");
+  public void testInit() {
```

Comment:
Not sure if this test is really needed, it doesn't assert anything. I understand it makes sure that "init" doesn't throw any exceptions, but still it is not clear about what it's really validating.